docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -2342,11 +2340,13 @@ function getCallback(asyncResult) {
   var text = asyncResult.value.data;
   var prop = asyncResult.value.sourceProperty;
 
-  Office.context.mailbox.item.setSelectedDataAsync('Setting ' + prop + ': ' + text, {}, setCallback);
-}
+  // Handle where Outlook on the web erroneously returns "null" instead of empty string.
+  if (Office.context.mailbox.diagnostics.hostName === 'OutlookWebApp'
+      && asyncResult.value.endPosition === asyncResult.value.startPosition) {
```

Comment:
This seems like a complicated way to check for null or an empty string. Is `value` a string?

Reply:
I'm not checking for a null or empty string. Instead, here we're checking that nothing has been selected so return an empty string.